### ISSUE TITLE
Mods

### ISSUE THREAD

**Issue by hamid12121** [AOF-Dev/MCinaBox#440]
Some mods like Liquidbounce not working 
Why ?

**Comment by gamecss** [comment]
Please test mod on your computer first

**Comment by hamid12121** [comment]
It's working fine in my pc

**Comment by gamecss** [comment]
国人就不用英语了

**Comment by gamecss** [comment]
你安装的模组api是和电脑一样的？

**Comment by gamecss** [comment]
Are you installing the same mod loader as on your computer?

**Comment by gamecss** [comment]
这应该和启动器java gl lwjgl al没什么关系？

**Comment by hamid12121** [comment]
Yes it's liquid bounce you can search in the web and copy it in mods in minecraft pc and it will work good but not running in this

**Comment by hamid12121** [comment]
I installed Wolfram version 1.2 in my pc and I copied %appdata%/.minecraft/version/Wolfram MC 1.2 folder but it's crashing ...
I have copied all of .minecraft folder in gamedir and it's working fine 
But mods not working ... 
Why

**Comment by gamecss** [comment]
这。。

**Comment by gamecss** [comment]
mcinabox用的依赖库和电脑不一样
可能和mod不工作有一讷讷关系

**Comment by gamecss** [comment]
Mcinabox uses a different dependency library than a computer
Maybe it has something to do with MOD not working
